docs/resources/rts_software_config_v1.md

```diff
@@ -22,24 +22,39 @@ resource "flexibleengine_rts_software_config_v1" "myconfig" {
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the software configuration.
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the RTS software resource.
+  If omitted, the provider-level region will be used. Changing this will create a new RTS software resource.
 
-* `group` - (Optional) The namespace that groups this software configuration by when it is delivered to a server.
+* `name` - (Required, String, ForceNew) The name of the software configuration.
```

Comment:
```suggestion
* `name` - (Required, String, ForceNew) The name of the software configuration. Changing this will create a new RTS software resource.
```